pkgs/cuda-packages/libcusolver/fixup.nix

```diff
@@ -1,3 +1,15 @@
 # NOTE: All fixups must be at least binary functions to avoid callPackage adding override attributes.
-{ lib, libcublas }:
-prevAttrs: { buildInputs = prevAttrs.buildInputs or [ ] ++ [ (lib.getLib libcublas) ]; }
+{ lib
+, libcublas
+, libcusparse
+, libnvjitlink
```

Comment:
I can't remember if `libnvjitlink` was available on JetPack 5 -- if it isn't (or really, if any attributes aren't present) and this expression is `callPackage`-d, it will die due to missing attributes. You might have to supply a default (`? null`) and add checks which mark the package as broken or unavailable depending on whether all the requisite packages are available.